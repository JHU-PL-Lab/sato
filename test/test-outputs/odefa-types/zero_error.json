[
  {
    "err_errors": [
      [
        "Error_binop",
        {
          "err_binop_left_aliases": [ "c", "b" ],
          "err_binop_right_aliases": [],
          "err_binop_left_val": "0",
          "err_binop_right_val": "0",
          "err_binop_operation": "0 <> 0"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "d = a / c",
    "err_steps": 149
  }
]