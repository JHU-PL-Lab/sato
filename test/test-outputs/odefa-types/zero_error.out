Type errors for:
- Input sequence  : []
- Found at clause : d = a / c
--------------------
* Left Val   : c = b = 0
* Constraint : 0 <> 0
1 errors found. No further control flows exist.
